### ISSUE TITLE
Using nested filter primitives

### ISSUE THREAD

**Issue by MentalGear** [mkaemmerer/filter-effects#7]
I'm wondering how filter primitives that have a second level should be handeled in the js object notion. What would be the correct way to use nested filter primitives like feDistantLight with filter-effects, or are they not supported ?

```
    <feDiffuseLighting
      surfaceScale="0.8"
      diffuseConstant="1.5"
      lighting-color="#ffffff"
      in="turbulence"
      result="diffuseLighting"
    >
      <feDistantLight azimuth="100" elevation="17" />
    </feDiffuseLighting>
```

**Comment by mkaemmerer** [comment]
Good question! I haven't written documentation for how to use nested filters, because I think the API I have for it right now is kind of bad honestly 😅 . That said, there *is* technically a way to nest filters with the current 0.9.0 release. You can add them by including a `children` key in the object notation.

Check out the discussion in https://github.com/mkaemmerer/filter-effects/issues/1#issuecomment-495963213 for some examples.

**Comment by mkaemmerer** [comment]
Also, if you have ideas for what a more ergonomic API for nested filters would look like, let me know. I'd be happy to make some improvements!

**Comment by MentalGear** [comment]
Yes, I figured this out by exploring the source code. Anyways, thank you @mkaemmerer .
